modules/runner/src/integrationTest/java/org/apache/ignite/internal/rest/authentication/ItAuthenticationTest.java

```diff
@@ -107,7 +107,38 @@ public void disabledAuthentication(TestInfo testInfo) {
     }
 
     @Test
-    public void changeCredentials(TestInfo testInfo) throws InterruptedException {
+    public void defaultUser(TestInfo testInfo) throws InterruptedException, IOException {
+        // when
```

Comment:
Here you use absolutely different notation. Should be `// When.`, right? The goal is to have uniformed approach